src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -165,13 +158,13 @@ describe('manage nsfs cli bucket flow', () => {
         });
 
         it('cli delete bucket and delete temp dir', async () => {
-            let is_path_exists = await native_fs_utils.is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
-            expect(is_path_exists).toBe(true);
+            let path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
```

Comment:
I would change the name of the function instead of the name of the variable, for example:
```suggestion
            let is_path_exists = await check_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
```

Reply:
Liran asked Naveen to change it to this name a few PRs ago, and I also prefer is_... as predicates names

Reply:
- I saw that the function was changed from `config_file_exists()` to `is_path_exists()` in PR #7718 since the first name was not accurate to the function functionality.
- Adding a verb ('check_'/'validate_', etc.) instead of the 'is_' in the name of the function is just a matter of style.